pkg/acsengine/defaults.go

```diff
@@ -82,6 +82,14 @@ func setOrchestratorDefaults(cs *api.ContainerService) {
 		if a.OrchestratorProfile.KubernetesConfig.NetworkPolicy == "" {
 			a.OrchestratorProfile.KubernetesConfig.NetworkPolicy = DefaultNetworkPolicy
 		}
+		if a.OrchestratorProfile.KubernetesConfig.ClusterSubnet == "" {
+			if a.OrchestratorProfile.IsVNETIntegrated() {
+				// When VNET integration is enabled, all masters, agents and pods share the same large subnet.
```

Comment:
What do you think about either of the following to cut down on nested conditionals:

1. refactor into a function, e.g. something like `vnetIsIntegrated()`
1. multiple conditions
```golang
		if a.OrchestratorProfile.KubernetesConfig.ClusterSubnet == "" && a.OrchestratorProfile.IsVNETIntegrated() {
			// When VNET integration is enabled, all masters, agents and pods share the same large subnet.
			a.OrchestratorProfile.KubernetesConfig.ClusterSubnet = DefaultKubernetesSubnet
		} else {
			a.OrchestratorProfile.KubernetesConfig.ClusterSubnet = DefaultKubernetesClusterSubnet
		}
```